libr/core/panels.c

```diff
@@ -4208,29 +4223,29 @@ static RPanels *get_cur_panels(RPanelsRoot *panels_root) {
 static bool handle_tab(RCore *core) {
 	r_cons_gotoxy (0, 0);
 	if (core->panels_root->n_panels <= 1) {
-		r_cons_printf (R_CONS_CLEAR_LINE"%s[Tab] t:new -:del"Color_RESET, core->cons->context->pal.graph_box2);
+		r_cons_printf (R_CONS_CLEAR_LINE"%s[Tab] t:new -:del =:name"Color_RESET, core->cons->context->pal.graph_box2);
 	} else {
 		int min = 1;
 		int max = core->panels_root->n_panels;
-		r_cons_printf (R_CONS_CLEAR_LINE"%s[Tab] [%d..%d]:select; p:prev; n:next; t:new -:del"Color_RESET, core->cons->context->pal.graph_box2, min, max);
+		r_cons_printf (R_CONS_CLEAR_LINE"%s[Tab] [%d..%d]:select; p:prev; n:next; t:new -:del =:name"Color_RESET, core->cons->context->pal.graph_box2, min, max);
 	}
 	r_cons_flush ();
 	int ch = r_cons_readchar ();
 	if (handle_tab_nth (core, ch)) {
 		return true;
 	}
 	switch (ch) {
-	case 'n':
-		if (handle_tab_next (core)) {
-			return true;
-		}
-		break;
-	case 'p':
-		return handle_tab_prev (core);
-	case '-':
-		return handle_tab_del (core);
-	case 't':
-		return handle_tab_new (core);
+		case 'n':
+			return handle_tab_next (core);
```

Comment:
Wrong indent

Reply:
lol it's not wrong imho but okay if you say so. hang on.